README.md

```diff
@@ -13,6 +13,7 @@ helmfile.
   If you can't access it, then you do not have access to private repositories(Goto previous step) and check with Devops
   team. If you can access, then create a branch something like 'adding-permissions-your-name'.
 - A local clone of the following repositories [wa-standalone-task-bpmn](https://github.com/hmcts/wa-standalone-task-bpmn) & [wa-task-configuration-template](https://github.com/hmcts/wa-task-configuration-template)
+- The master branch must be used with Mac M1, M2 chip. For Intel chip, use the branch 'https://github.com/hmcts/wa-kube-environment/tree/kube-env-mac_intel_chips'.
```

Comment:
Should probably re-phrase this since there is now an M4 chip and there will probably be more!
E.g.
`This environment setup is intended to be used with Apple Mac machines running Apple silicone such as M4 / M4 Max.  For Intel chips .....`